### DISCUSSION TITLE
馒头添加站点正确但是无法下载种子

### DISCUSSION THREAD

**Discussion by syyee** [jxxghp/MoviePilot#2644] [问题咨询]
如图，1.9.14版本，馒头站点应该配置对了，测试通过能搜索、能浏览。日志中下载种子抓到的地址也是对的（用浏览器打开种子就下来了）。可是MP就是下载失败提示无法打开链接。。。。
![image](https://github.com/user-attachments/assets/33e2e91b-5259-4251-968d-021200eee19e)

**Comment by dengtongcai** [comment]
同样的问题

**Reply by wodty** [reply]
mp下载目录跟qb下载目录不一致或者没权限

**Comment by MoogleXX** [comment]
感觉有时候是馒头问题。我遇到几个这种情况直接去馒头点地址也打不开。有些又能MP下载。

**Reply by syyee** [reply]
现在遇到的是直接地址都可以打开，mp里抓到的种子地址也是都对的。只是mp提示无法打开链接，下载失败。怀疑是不是download.py下载馒头种子链接有啥错误

**Reply by msw1990** [reply]
我也是同样的问题，不过不过MP日志里抓去的种子地址我用浏览器也打不开，并不是馒头种子的域名

**Comment by AJHarris-Large** [comment]
1.9.17，同样的问题

**Reply by AJHarris-Large** [reply]
> 1.9.17，同样的问题
现在是织梦和其他PT站可以正常下载，只有每次解析MT的种子时候出现这个问题

**Comment by longfei12k** [comment]
这个去控制面板哪里，勾选下测试通过域名，试试。

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年10月18日 23:06 |
| 收件人 | jxxghp/MoviePilot ***@***.***> |
| 抄送人 | Subscribed ***@***.***> |
| 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |

我也是同样的问题，不过不过MP日志里抓去的种子地址我用浏览器也打不开，并不是馒头种子的域名

**Comment by msw1990** [comment]
好的，我试试看



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年10月18日(周五) 晚上11:17
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644)




 这个去控制面板哪里，勾选下测试通过域名，试试。
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&gt; |
 | 发送日期 | 2024年10月18日 23:06 |
 | 收件人 | jxxghp/MoviePilot ***@***.***&gt; |
 | 抄送人 | Subscribed ***@***.***&gt; |
 | 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |
 
 我也是同样的问题，不过不过MP日志里抓去的种子地址我用浏览器也打不开，并不是馒头种子的域名

**Comment by msw1990** [comment]
问一下，是馒头的控制台吗？&nbsp; 我好像没找到相关设置



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年10月18日(周五) 晚上11:17
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644)




 这个去控制面板哪里，勾选下测试通过域名，试试。
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&gt; |
 | 发送日期 | 2024年10月18日 23:06 |
 | 收件人 | jxxghp/MoviePilot ***@***.***&gt; |
 | 抄送人 | Subscribed ***@***.***&gt; |
 | 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |
 
 我也是同样的问题，不过不过MP日志里抓去的种子地址我用浏览器也打不开，并不是馒头种子的域名

**Comment by longfei12k** [comment]
控制台，看看有没有域名选项，没有估计得提交管理组了。

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年10月18日 23:29 |
| 收件人 | jxxghp/MoviePilot ***@***.***> |
| 抄送人 | longfei12k ***@***.***>,
Comment ***@***.***> |
| 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |

问一下，是馒头的控制台吗？&nbsp; 我好像没找到相关设置



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年10月18日(周五) 晚上11:17
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644)




这个去控制面板哪里，勾选下测试通过域名，试试。

---- 回复的原邮件 ----
| 发件人 | ***@***.***&gt; |
| 发送日期 | 2024年10月18日 23:06 |
| 收件人 | jxxghp/MoviePilot ***@***.***&gt; |
| 抄送人 | Subscribed ***@***.***&gt; |
| 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |

我也是同样的问题，不过不过MP日志里抓去的种子地址我用浏览器也打不开，并不是馒头种子的域名

**Comment by msw1990** [comment]
试了一下，可以不选第三方种子服务器的域名。但是福建联通连不上默认的种子服务器，也是下载不了，裂开，我试试改host&nbsp; 应该是找到原因了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年10月18日(周五) 晚上11:33
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644)




 控制台，看看有没有域名选项，没有估计得提交管理组了。
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&gt; |
 | 发送日期 | 2024年10月18日 23:29 |
 | 收件人 | jxxghp/MoviePilot ***@***.***&gt; |
 | 抄送人 | longfei12k ***@***.***&gt;,
 Comment ***@***.***&gt; |
 | 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |
 
 问一下，是馒头的控制台吗？&amp;nbsp; 我好像没找到相关设置
 
 
 
 ---原始邮件---
 发件人: ***@***.***&amp;gt;
 发送时间: 2024年10月18日(周五) 晚上11:17
 收件人: ***@***.***&amp;gt;;
 抄送: ***@***.******@***.***&amp;gt;;
 主题: Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644)
 
 
 
 
 这个去控制面板哪里，勾选下测试通过域名，试试。
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&amp;gt; |
 | 发送日期 | 2024年10月18日 23:06 |
 | 收件人 | jxxghp/MoviePilot ***@***.***&amp;gt; |
 | 抄送人 | Subscribed ***@***.***&amp;gt; |
 | 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |
 
 我也是同样的问题，不过不过MP日志里抓去的种子地址我用浏览器也打不开，并不是馒头种子的域名

**Comment by msw1990** [comment]
还是搞不定，奇怪了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年10月18日(周五) 晚上11:33
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644)




 控制台，看看有没有域名选项，没有估计得提交管理组了。
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&gt; |
 | 发送日期 | 2024年10月18日 23:29 |
 | 收件人 | jxxghp/MoviePilot ***@***.***&gt; |
 | 抄送人 | longfei12k ***@***.***&gt;,
 Comment ***@***.***&gt; |
 | 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |
 
 问一下，是馒头的控制台吗？&amp;nbsp; 我好像没找到相关设置
 
 
 
 ---原始邮件---
 发件人: ***@***.***&amp;gt;
 发送时间: 2024年10月18日(周五) 晚上11:17
 收件人: ***@***.***&amp;gt;;
 抄送: ***@***.******@***.***&amp;gt;;
 主题: Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644)
 
 
 
 
 这个去控制面板哪里，勾选下测试通过域名，试试。
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&amp;gt; |
 | 发送日期 | 2024年10月18日 23:06 |
 | 收件人 | jxxghp/MoviePilot ***@***.***&amp;gt; |
 | 抄送人 | Subscribed ***@***.***&amp;gt; |
 | 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |
 
 我也是同样的问题，不过不过MP日志里抓去的种子地址我用浏览器也打不开，并不是馒头种子的域名

**Comment by longfei12k** [comment]
看来也只有我能给你解决了

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年10月19日 00:00 |
| 收件人 | jxxghp/MoviePilot ***@***.***> |
| 抄送人 | longfei12k ***@***.***>,
Comment ***@***.***> |
| 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |

还是搞不定，奇怪了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年10月18日(周五) 晚上11:33
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644)




控制台，看看有没有域名选项，没有估计得提交管理组了。

---- 回复的原邮件 ----
| 发件人 | ***@***.***&gt; |
| 发送日期 | 2024年10月18日 23:29 |
| 收件人 | jxxghp/MoviePilot ***@***.***&gt; |
| 抄送人 | longfei12k ***@***.***&gt;,
Comment ***@***.***&gt; |
| 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |

问一下，是馒头的控制台吗？&amp;nbsp; 我好像没找到相关设置



---原始邮件---
发件人: ***@***.***&amp;gt;
发送时间: 2024年10月18日(周五) 晚上11:17
收件人: ***@***.***&amp;gt;;
抄送: ***@***.******@***.***&amp;gt;;
主题: Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644)




这个去控制面板哪里，勾选下测试通过域名，试试。

---- 回复的原邮件 ----
| 发件人 | ***@***.***&amp;gt; |
| 发送日期 | 2024年10月18日 23:06 |
| 收件人 | jxxghp/MoviePilot ***@***.***&amp;gt; |
| 抄送人 | Subscribed ***@***.***&amp;gt; |
| 主题 | Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644) |

我也是同样的问题，不过不过MP日志里抓去的种子地址我用浏览器也打不开，并不是馒头种子的域名

**Comment by raycrystal** [comment]
1.9.17，同样的问题，怀疑馒头的种子服务器是不是返回值比较特殊被认为无法下载

**Comment by 924274383** [comment]
是这样，同福建联通馒头。。。

**Comment by snoopy4444cn** [comment]
你们有没有试过把ipv6给关了？

**Comment by suzukua** [comment]
同样的问题，因为馒头返回了302跳转google。用curl -ivv 就可以复现

**Comment by wodty** [comment]
国内有些区域不行，福州网络无法链接到馒头。厦门的可以




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jxxghp/MoviePilot"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年2月8日(星期六) 上午10:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [jxxghp/MoviePilot] 馒头添加站点正确但是无法下载种子 (Discussion #2644)





 
同样的问题，因为馒头返回了302跳转google。用curl -ivv 就可以复现

**Comment by King-hsw** [comment]
这问题持续这么久了么，一年了还没解决。。

**Reply by wodty** [reply]
地区网问题，上梯子了

**Reply by King-hsw** [reply]
> 地区网问题，上梯子了
有设置代理地址，但是下载的时候不会走代理么

**Reply by wodty** [reply]
不会啊，访问的时候走代理，下载种子会。

**Reply by wodty** [reply]
不会啊，访问的时候走代理，下载种子会。下载内容不会

**Reply by King-hsw** [reply]
> 不会啊，访问的时候走代理，下载种子会。下载内容不会

https://api.m-team.cc/api/torrent/genDlToken 这一步应该还在下种子吧，很奇怪，本地电脑用postman调，没问题。同一个局域网下的NAS，在mp上可视化点就不行
